vm/policy.go

```diff
@@ -63,17 +106,16 @@ func PolicyHandling(t string, path string, o PolicyOptions) error {
 		return err
 	}
 
-	client := pb.NewPolicyServiceClient(conn)
-
-	req := pb.Policy{
-		Policy: policyEventData,
-	}
+	if isNonK8sEnv {
+		// Non-K8s control plane with kvmservice, hence send policy over HTTP
+		if err = sendPolicyOverHTTP(httpAddress, policyEventData); err != nil {
+			return err
+		}
+	} else {
+		// Systemd mode, hence send policy over gRPC
+		if err = sendPolicyOverGRPC(o, policyEventData); err != nil {
+			return err
```

Comment:
Like I mentioned in flags perhaps naming the flag control plane or something would be better rather than Non k8s.

Reply:
Also maybe we can have additional checking like if we have a gRPC flag passed down then don't bother checking the flag.

Reply:
Since we are using kvm-service, we can name that flag as kvms instead of non-k8s. That will be different. Please suggest